resources/master_term_bank_en.csv

```diff
@@ -19,4 +19,5 @@ application/json,igmi,"An acronym for I'm going to make it.",en,GLOSSETA,general
 application/json,dao,"An acronym for decentralized autonomous organizations.  DAO's are distributed internet communities that self-organize around a common niche and are driven by purpose oriented tokens created on blockchain protocols.  Developer DAO (D_D) is an example of a DAO.",en,GLOSSETA,general
 application/json,glosseta,"Glosseta is a open-source web3 glossary meant to help the community understand the terms/lingo floating around the internet.  Glosseta is meant to be your in pocket companion for your web3 journey and grow as the overall community grows. We're all going to make it (WAGMI)!",en,GLOSSETA,application
 application/json,probably nothing,"Definitely something.",en,GLOSSETA,general
-application/json,gas,"Gas refers to the unit that measures the amount of computational effort required to execute specific operations on the Ethereum network. Since each Ethereum transaction requires computational resources to execute, each transaction requires a fee. Gas refers to the fee required to conduct a transaction on Ethereum successfully.",en,GLOSSETA,general
\ No newline at end of file
+application/json,gas,"Gas refers to the unit that measures the amount of computational effort required to execute specific operations on the Ethereum network. Since each Ethereum transaction requires computational resources to execute, each transaction requires a fee. Gas refers to the fee required to conduct a transaction on Ethereum successfully.",en,GLOSSETA,general
+application/json,EIP,"Ethereum Improvement Proposal. As ethereum is a decentralize blockchain, community can propose changes then implement them after consensus on them",en,GLOSSETA,protocol
```

Comment:
Hi thank you so much for adding this term to the knowledge bank!   A couple of things:

- Let's make `EIP` lowercase as that's how i've been uploading the terms.  On the frontend they get uppercased
- Slight change proposal to the definition:  

```
An acronym for Ethereum Improvement Proposal.  EIPs describe how the Ethereum blockchain works through core protocol specifications, client APIs, and contract standards. As Ethereum is an open and decentralized blockchain, the community can propose changes through an EIP then implement them after consensus is reached.
```


Also for the future, could you please create an issue for any new definition intake requests using this [template](https://github.com/narbs91/glosseta/issues/new?assignees=&labels=definition&template=definition-request.yml&title=%5BDefinition+Request%5D%3A+)?  Don't worry about it this time but it just helps me keep a history what's come through and act as a means of organization.  Thanks again for contributing, much appreciated :)

Reply:
Also i just pushed a fix that corrected some grammar and spelling; however, as a result i had to change up the category from `GLOSSETA` to 'GLOSSETA-PROD` and added new term as well.  So you'll have to resolve those conflicts as well.  Sorry about that but we shouldn't need to make a change like this anymore and if we have to make updates we can use the timestamp on the upload to take the latest entry.  There will be a fix to address that at a later time